python/tests/conftest.py

```diff
@@ -118,10 +149,57 @@ def moto_s3_uri_incl_bucket(moto_s3_endpoint_and_credentials):
     ] + ":" + bucket + "?access=" + aws_access_key + "&secret=" + aws_secret_key + "&port=" + port
 
 
-@pytest.fixture(scope="function", params=("S3", "LMDB"))
-def arctic_client(request, moto_s3_uri_incl_bucket, tmpdir):
+@pytest.fixture
+def azurite_azure_test_connection_setting(azurite_port, spawn_azurite):
+    global bucket_id
+
+    container = f"testbucket{bucket_id}"
+    bucket_id = bucket_id + 1
+
+    credential_name = "devstoreaccount1"
+    credential_key = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
+    endpoint = f"127.0.0.1:{azurite_port}"
+    is_https = False
+    connect_to_azurite = True
+
+    return endpoint, container, credential_name, credential_key, is_https, connect_to_azurite
```

Comment:
Minor: It will be much easier to read and maintain if you return an object here with these fields instead of a random tuple of values.

You can either return a `dict` or use the `arcticc.pb2.azure_storage_pb2.Config` class which seems to have these fields.

Reply:
Closed per discussion with Qi. It will be addressed in bulk in a separate PR.